consensus/src/quorum_store/tests/direct_mempool_quorum_store_test.rs

```diff
@@ -66,11 +67,9 @@ async fn test_block_request_no_txns() {
             assert!(payload.is_empty());
             match payload {
                 Payload::DirectMempool(txns) => assert!(txns.is_empty()),
```

Comment:
I don't think we need this case here, given payload is empty.

Reply:
Agree.